src/sender.js

```diff
@@ -121,6 +125,11 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue }) {
       await sendToQueue(failedTx)
     }
     ackMsg(msg)
+
+    if (insufficientFunds) {
+      channel.close()
+      waitForFunds(web3Instance, VALIDATOR_ADDRESS, initialize)
```

Comment:
I think it would be useful to add 2 logs. One with a message that it will stop processing jobs because of the funds, and a second log with a message that the validator have funds again and it will start processing jobs

Reply:
Added.